data/reusables/actions/jobs/section-using-conditions-to-control-job-execution.md

```diff
@@ -1,5 +1,9 @@
 You can use the `jobs.<job_id>.if` conditional to prevent a job from running unless a condition is met. {% data reusables.actions.if-supported-contexts %}
 
+{% note %}
+**Note:** This if condition applies before `jobs.<job_id>.matrix`.
+{% endnote %}
```

Comment:
Not rendering correctly in the preview (https://docs-27169-1523dd.preview.ghdocs.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif). Usually we have blank lines after the opening tag and before the closing tag. I'll see if this fixes it.

```suggestion
{% note %}

**Note:** This if condition applies before `jobs.<job_id>.matrix`.

{% endnote %}
```